src/styles/utils/getAutoCompleteSuggestionContainerStyle/index.ts

```diff
@@ -0,0 +1,32 @@
+import type {ViewStyle} from 'react-native';
+// eslint-disable-next-line no-restricted-imports
+import shouldPreventScrollOnAutoCompleteSuggestion from '@styles/utils/autoCompleteSuggestion';
+import CONST from '@src/CONST';
+
+const shouldPreventScroll = shouldPreventScrollOnAutoCompleteSuggestion();
+
+/**
+ * Gets the correct position for auto complete suggestion container
+ */
+export default function getAutoCompleteSuggestionContainerStyle(
+    itemsHeight: number,
+    shouldBeDisplayedBelowParentContainer: boolean,
+    isEditComposer: boolean,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    _containerHeight: number,
+): ViewStyle {
+    'worklet';
+
+    const borderWidth = 2;
```

Comment:
We could also move this to `styles/variables`, which would allow us to use it in `src/libs/SuggestionUtils.ts` instead of a hardcoded number